Sources/FigmaExport/Loaders/ColorsLoader.swift

```diff
@@ -13,34 +13,60 @@ final class ColorsLoader {
         self.figmaParams = figmaParams
         self.colorParams = colorParams
     }
-    
-    func load() throws -> (light: [Color], dark: [Color]?) {
-        if let useSingleFile = colorParams?.useSingleFile, useSingleFile {
-            return try loadColorsFromSingleFile()
-        } else {
+
+    func load() throws -> (light: [Color], dark: [Color]?, lightHC: [Color]?, darkHC: [Color]?) {
+        guard let useSingleFile = colorParams?.useSingleFile, useSingleFile else {
             return try loadColorsFromLightAndDarkFile()
         }
+        return try loadColorsFromSingleFile()
     }
 
-    private func loadColorsFromLightAndDarkFile() throws -> (light: [Color], dark: [Color]?) {
+    private func loadColorsFromLightAndDarkFile() throws -> (light: [Color],
+                                                             dark: [Color]?,
+                                                             lightHC: [Color]?,
+                                                             darkHC: [Color]?) {
         let lightColors = try loadColors(fileId: figmaParams.lightFileId)
         let darkColors = try figmaParams.darkFileId.map { try loadColors(fileId: $0) }
-        return (lightColors, darkColors)
+        let lightHighContrastColors = try figmaParams.lightHighContrastFileId.map { try loadColors(fileId: $0) }
+        let darkHighContrastColors = try figmaParams.darkHighContrastFileId.map { try loadColors(fileId: $0) }
+        return (lightColors, darkColors, lightHighContrastColors, darkHighContrastColors)
     }
 
-    private func loadColorsFromSingleFile() throws -> (light: [Color], dark: [Color]?) {
+    private func loadColorsFromSingleFile() throws -> (light: [Color],
+                                                       dark: [Color]?,
+                                                       lightHC: [Color]?,
+                                                       darkHC: [Color]?) {
         let colors = try loadColors(fileId: figmaParams.lightFileId)
         let darkSuffix = colorParams?.darkModeSuffix ?? "_dark"
+        let lightHCSuffix = colorParams?.lightHCModeSuffix ?? "_lightHC"
+        let darkHCSuffix = colorParams?.darkHCModeSuffix ?? "_darkHC"
+
         let lightColors = colors
             .filter { !$0.name.hasSuffix(darkSuffix) }
+            .filter { !$0.name.hasSuffix(lightHCSuffix) }
+            .filter { !$0.name.hasSuffix(darkHCSuffix) }
```

Comment:
Combine three filter operations in to the one to avoid iterating over array three times.